how_to_edit_photo_image_programmaticaly.md

```diff
@@ -0,0 +1,166 @@
+---
+title: How to edit your photo or image programmatically
+description: Library for image editing. Photo editing automation. Image manipulation by NET (C#) program. Removing photo background. Online photo editor.
+keywords: [edit a photo, editing of pictures, edit an image, photo background editor, photo background change, resizing an image, resizing photos, photo sizing, picture resizer, image crop, crop photo, filter photo, online photo editor, online image editor, photo editing app, free photo editor]
+---
+
+# How to edit your photo or image programmatically
+
+## Intro
+
+In real life, we pay attention to colorful images, and in the virtual world, our perception inevitably reacts to a beautiful picture. The more attractive the shop window, the more likely we will go into the store itself; how the more interesting the visual content of the site, whether it is an online store or a thematic blog, the higher its traffic. However, the search and integration of engaging pictures is only part of the task facing the author, who forms the visual approach of the page. The next, and no less important step is editing photos and images. Properly edited photos - the same quality mark of the site as its text content. It increases website attractiveness, and user conversion rate, as a result creating new business opportunities for commercial sites and raising the recognition of researchers. The program library for photo editing is intended to facilitate the rapid development of new applications and services.
```

Comment:
@anms23 , maybe article should be formatted for end user?

Reply:
It is a markdown format. You could see the article html formatting for end-users on the github pages: 
https://aspose-imaging.github.io/aspose-imaging-probe/how_to_edit_photo_image_programmaticaly

Reply:
@anms23 , yes I know it, I mean that article text should be aligned by width, text indents for first lines etc, like we do in usual articles, is not it?

Reply:
@sam199008 , I've added indents and aligned text.
Main page: https://aspose-imaging.github.io/aspose-imaging-probe/how_to_edit_photo_image_programmaticaly

Reply:
@sam199008 , removed.